.github/workflows/lint.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: psf/black@stable
         with:
-          options: "--check --verbose"
```

Comment:
The `verbose` option lists files that did not cause any failure - it seems to me that it only create noise when looking for failures, but I'd be happy to put it back if that's the preferred configuration.